packages/devtools_app/lib/src/memory/flutter/memory_controller.dart

```diff
@@ -48,6 +51,41 @@ class MemoryController extends DisposableController
 
   static const String logFilenamePrefix = 'memory_log_';
 
+  bool showHeatMap = true;
+
+  final List<Snapshot> snapshots = [];
+
+  Snapshot get lastSnapshot => snapshots.isNotEmpty ? snapshots.last : null;
```

Comment:
we have a `safeLast` extension method that you can call here to save a couple characters.

Reply:
Done